mgmt/ProcessManager.cc

```diff
@@ -243,7 +243,28 @@ ProcessManager::signalManager(int msg_id, const char *data_raw, int data_len)
   mh->msg_id   = msg_id;
   mh->data_len = data_len;
   memcpy((char *)mh + sizeof(MgmtMessageHdr), data_raw, data_len);
+  this->signalManager(mh);
+}
+
+void
+ProcessManager::signalManager(int msg_id, std::string_view text)
```

Comment:
Where is this being called?

Reply:
No where, since it didn't exist. I do plan to use it later in another PR I am working on (for #1971). Unless we want to do a massive rewrite of most of the code, it will be necessary to do updates like this to support a view based interface before it's actually used.